tensorflow/contrib/makefile/README.md

```diff
@@ -142,6 +142,11 @@ xcode-select --install
 If this is a new install, you will need to run XCode once to agree to the
 license before continuing.
 
+If you do not have [Homebrew](http://brew.sh/) installed, you will need to install via terminal by using:
```

Comment:
I think it would be better to just tell people to install Homebrew using their instructions, instead of duplicating it here, which might go stale.